packages/manager/modules/pci/src/projects/project/data-processing/data-processing.controller.js

```diff
@@ -26,6 +28,10 @@ export default class {
 
   $onInit() {
     this.subscribeToMessages();
+    this.atInternet.trackPage({
+      name: 'public-cloud::pci::projects::project::data-processing',
+      level2: '11',
```

Comment:
No need to specify `level2` each time (especially it could be different across regions) it is already injected each time https://github.com/ovh/manager/blob/master/packages/manager/apps/public-cloud/src/components/at-internet/at-internet.constant.js#L3

Reply:
Commit amended without those, then. Thanks!